src/runtime/hexagon_remote/halide_hexagon_remote.idl

```diff
@@ -4,11 +4,12 @@ interface halide_hexagon_remote
     typedef unsigned long handle_t;
 
     // Routine to initialize a module on the remote side. Returns a pointer to the module.
-    long initialize_kernels(in buffer code, rout handle_t module_ptr);
+    long initialize_kernels(in buffer code, in long use_dlopen, in long use_dlopenbuf,
```

Comment:
Makes sense.
I will attack this one in the morning, so not in this push, but next one.